src/controllers/update/secure.update.filter.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { NextFunction, Request, Response } from "express";
+
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { IsSecureRegisterKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.SECURE_REGISTER_FILTER_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    const notLiveTitle: string = "(NOT LIVE)";
```

Comment:
Only (Not Live) or are we adding that to something else?

Reply:
I see the view is rendering whatever is in title now?

Reply:
I figured it was easier to just append that part to the title. That way, I wouldn't have to add anything in the register journey secure register controller. The view is rendering the title set on the view itself. If it GET request comes from the register journey controller then it wont append the `notLiveTitle` string to the title. Which means the page title will not look any different in the register journey.

Reply:
Should we pass a notLiveTitle == "" in the controller for register? Feels wrong to rely on it not being defined. Also renaming notLiveTitleExtension may help clarify that this is an extension to and not the title itself.

Reply:
So pass in titleExtension in both controllers?

Reply:
`titleExtension` is definitely a better name. I was hoping someone had a nicer alternative to `notLiveTitle`.